docs/pages/workflow/expo-cli.md

```diff
@@ -70,42 +71,40 @@ The default UI has a QR code and a list of keyboard shortcuts you can press, the
+- `o`: Open project code in your editor. This can be configured with the `EXPO_EDITOR` and `EDITOR` [environment variables](#environment-variables).
 - `c`: Show development server URL as a QR code in the terminal.
 - `?`: Show all Terminal UI commands.
 
 ### Server URL
 
-The URL is served over a LAN connection that utilizes the local network by default. You can change this behavior to localhost-only by using the flag `--host localhost` or `--localhost`.
+By default, the project is served over a LAN connection. You can change this behavior to localhost-only by using the flag `npx expo start --localhost`.
 
-- `--port`: Port to start the dev server on (does not apply to Webpack or `--tunnel` URLs). Default: **19000**.
-- `--https`: Start the dev server using a secure origin. This is currently only supported in Webpack.
+Other available options are:
 
-You can force the URL to be whatever you like by using the `EXPO_PACKAGER_PROXY_URL` environment variable.
+- `--port`: Port to start the dev server on (does not apply to Webpack or [tunnel URLs](#tunneling)). Default: **19000**.
+- `--https`: Start the dev server using a secure origin. This is currently only supported on web.
 
-For example:
+You can force the URL to be any value with the `EXPO_PACKAGER_PROXY_URL` environment variable, for example:
 
 <Terminal cmd={[
 'export EXPO_PACKAGER_PROXY_URL=http://expo.dev',
 '$ npx expo start'
 ]} />
 
-Will open apps to: `exp://expo.dev:80` (the `:80` is a temporary workaround for Android support).
+Will open apps to: `exp://expo.dev:80` (the `:80` is a temporary workaround for Android websockets).
 
 #### Tunneling
 
-Sometimes you may find it difficult to connect your device to your machine. Many factors can cause this to happen: restrictive network conditions (common for public Wi-Fi), firewalls (common for Windows users), or Emulator misconfiguration.
-
-To combat these, we provide built-in URL "tunneling" with ngrok for free! This enables you to forward your `localhost` URL to a public URL that's accessible from anywhere in the world (more on the security implications later).
+Restrictive network conditions (common for public Wi-Fi), firewalls (common for Windows users), or Emulator misconfiguration can make it difficult to connect a remote device to your dev server over lan/localhost.
 
-This means your device simply needs to connect to the internet to access your dev server.
+Sometimes it's easier to connect to a dev server over a proxy URL that's accessible from any device with internet access, this is referred to as **tunneling**. `npx expo start` provides built-in support for **tunneling** via [ngrok][ngrok].
 
-To enable tunneling, you first need `@expo/ngrok` installed either globally (recommended) or locally in your project:
+To enable tunneling, first install `@expo/ngrok`:
```

Comment:
```suggestion
To enable tunneling, first install `@expo/ngrok` as global package:
```

Reply:
It can be installed either globally or locally, in this case I'm just keeping the setup simple and leaning on the installation command.

Reply:
Okay, wasn't aware of that. I'll remove this suggestion.